modules/integration/pages/ledger-quick-start.adoc

```diff
@@ -158,50 +157,158 @@ You are all caught up to block height 109
+
+==== How do I know if the node has caught up with the test net?
+
+Search the `Starting Rosetta API server` startup log. There will be a log entry that says `You are all caught up to block XX`. This message confirms that you are caught up with all blocks.
+
+==== How to persist synced blocks data?
+
+Mount the `/data` directory elsewhere.
+
+==== Is the Rosetta node versioned?
+
+Not yet. Before launch, when we push to the `dfinity/rosetta-api:latest` image, it's usually a major update that we'll announce in our communication channels beforehand.
+
+We'll soon implement nightly builds of the image, and CI will ensure it works before pushing. Other than `latest`, those images will also be tagged with the build date, so for more reproducibility, it's possible to use the image of a specific date tag rather than `latest`. We'll announce when nightly builds become available.
+
+==== How do I connect to the main net instead of the test net?
+
+Start `dfinity/rosetta-api` with `--help`, you can see some additional CLI arguments that can be passed. Among those there are `--canister-id` and `--ic-url` which can be used to configure the ledger destination. At the moment, they default to the test net.
+
+**Note**: The main net is not live yet; it will be live some time before the publicly announced date, and we'll push the updated image to point to the main net to ensure you can perform testing on the main net beforehand.
+
+=== ICPT-specific Rosetta API details
+
+==== How are accounts generated and verified?
+
+- Generate an ED25519 keypair.
+- The secret key is used for signing transactions.
+- The public key is used for generating a self-authenticating Principal ID. For more information, see: https://sdk.dfinity.org/docs/interface-spec/index.html#_principals.
+- The Principal ID is hashed to generate the account address.
+
+==== How to use the public key to generate its account address?
+
+- Call the `[/construction/derive](https://www.rosetta-api.org/docs/ConstructionApi.html#constructionderive)` endpoint with the hex-encoded 32-byte public key.
+- Call the `pub_key_to_address` function in the JavaScript SDK.
+
+==== How to verify the checksum of an account address?
+
+- After hex decoding, the first 4 bytes is the big-endian CRC32 checksum of the rest of the address.
+- Call `[address_from_hex](https://github.com/dfinity/rosetta-client#working-with-account-addresses)` in the JavaScript SDK. It returns and error if checksum doesn't match.
+- [Here](https://gist.github.com/TerrorJack/d6c79b33e5b5d0f5d52f3a2c5cdacc60) is a Java implementation of address validation logic.
+
+==== What is the `signature_type` / `curve_type` for ED25519?
+
+- `signature_type` is `"ed25519"`
+- `curve_type` is `"edwards25519"`
+
+==== What kinds of transactions may appear in a block, and what do they mean?
```

Comment:
```suggestion
==== What kinds of transactions can appear in a block, and what do they mean?
```

Reply:
I thought I changed that to "might." Guess not.